src/components/application_manager/src/resumption/resumption_sql_queries.cc

```diff
@@ -877,6 +909,20 @@ const std::string kSelectSubscriptions =
     "FROM `application` "
     "WHERE `appID` = ? AND `deviceID` = ?);";
 
+const std::string kSelectCountUserLocation =
+    "SELECT COUNT (`idApplication`) "
+    "FROM `applicationUserLocation` "
+    "WHERE `idApplication` = (SELECT `idApplication` "
+    "FROM `application` "
+    "WHERE `appID` = ? AND `deviceID` = ?);";
+
+const std::string kSelectUserLocation =
+    "SELECT `col`, `colspan`, `level`, `levelspan`, `row`, `rowspan` "
+    "FROM `applicationUserLocation` "
+    "WHERE `idApplication` = (SELECT `idApplication` "
+    "FROM `application` "
+    "WHERE `appID` = ? AND `deviceID` = ?);";
+
```

Comment:
```suggestion
    "SELECT `col`, `colspan`, `level`, `levelspan`, `row`, `rowspan` "
    "FROM `applicationUserLocation` INNER JOIN `application` ON `applicationUserLocation`.`idApplication` = `application`.`idApplication` "
    "WHERE `appID` = ? AND `deviceID` = ?);";

```

Reply:
fixed in bf0ce24216a70351394e14311bafcfac997606f6